pom.xml

```diff
@@ -15,7 +15,7 @@
     </modules>
 
     <properties>
-        <sofa.ark.version>2.2.1</sofa.ark.version>
+        <sofa.ark.version>2.2.3-SNAPSHOT</sofa.ark.version>
```

Comment:
next version should be 2.2.2, please set to 2.2.2-SNAPSHOT if you need to change the version